### DISCUSSION TITLE
Server side pagination in tables

### DISCUSSION THREAD

**Discussion by Maratinhos** [ToolJet/ToolJet#7903] [Q&A]
Hello.
Help me to figure out the problems.
I'm using the documentation to set up server-side pagination. And something goes wrong or not.

The data loads to the table after a few seconds. (from Postgres, my table has 18 000 000 rows, 3 columns.)
But I get some errors:

- [Component test_data - Enable next page button] - Invalid property detected: Expected a value of type `boolean`, but received: `""`.
- [Component test_data - Table data] - Invalid property detected: Expected an array value, but received: "".
- [Component test_data - Loading state] - Invalid property detected: Expected a value of type `boolean`, but received: `""`.
...

![image](https://github.com/ToolJet/ToolJet/assets/14004598/6975af36-3c2d-453c-ac8f-8af274f0acfa)

Need I just wait and ignore the errors? 

And I try to change the page, and nothing happens.

![image](https://github.com/ToolJet/ToolJet/assets/14004598/a60553e9-6d3a-4080-bce4-89cd5ea20452)

**Comment by Maratinhos** [comment]
Version 2.17.1

**Comment by ShazanRizvi** [comment]
Hey @Maratinhos have you tried upgrading to the latest version? If not could you please upgrade to the latest version.
NOTE: Before upgrading please take a backup of the DB.

You can also get help from community members by joining on Slack: https://tooljet.com/slack
